client/src/components/MentorItem.jsx

```diff
@@ -0,0 +1,53 @@
+import { React } from "react";
+
+import { Box, Grid, Typography, Link, Divider, Stack } from "@mui/material";
+
+import LinkedInIcon from "@mui/icons-material/LinkedIn";
+import LanguageIcon from "@mui/icons-material/Language";
+
+const MentorItem = ({ name, pronouns, linkedIn, websiteUrl, icon }) => {
+  return (
+    <Grid container direction="row" sx={{ margin: "15px" }}>
+      <Grid container item xs={4} alignItems="center">
+        <Box component="img" src={icon} sx={{ width: "90%" }} />
+      </Grid>
+      <Grid
+        container
+        item
+        xs={8}
+        sx={{ paddingLeft: "10px" }}
```

Comment:
Using the MUI framework, we can leverage some predefined (and uniform) theme spacing units with integers w/o a unit definition. 

https://mui.com/material-ui/customization/spacing/

Might be helpful to consider this when thinking about responsive spacing across the pages

Reply:
oooooooooo ok will take a look and change accordingly, tyyy